apps/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 # Add applications
+ADD_SUBDIRECTORY(DensifyPointCloud)
 ADD_SUBDIRECTORY(InterfaceCOLMAP)
 ADD_SUBDIRECTORY(InterfaceMetashape)
-ADD_SUBDIRECTORY(DensifyPointCloud)
+ADD_SUBDIRECTORY(InterfaceOpenMVG)
```

Comment:
This 2 interfaces are deprecated, pls remove them;
OpenMVG has its own interface to OpenMVS
VisualSFM has accuracy issues, should not be used